pkgs/development/python-modules/pre-commit-hook-ensure-sops/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, buildPythonPackage, applyPatches, fetchFromGitHub, ruamel-yaml }:
+
+buildPythonPackage rec {
+  pname = "pre-commit-hook-ensure-sops";
+  version = "1.1";
+
+  src = applyPatches {
```

Comment:
Why do we need to create another derivation that has the patches applied?

Reply:
Fixed.